File: 789-escape-the-ghosts/789-escape-the-ghosts.cpp
class Solution {
public:
    bool escapeGhosts(vector<vector<int>>& ghosts, vector<int>& target) {
        int cg = abs(target[0] - ghosts[0][0]) + abs(target[1] - ghosts[0][1]);
        for(auto c: ghosts) {
            cg = min(cg, abs(target[0] - c[0]) + abs(target[1] - c[1]));
        }
        return cg > abs(target[0]) + abs(target[1]);
    }
};

File: 789-escape-the-ghosts/789-escape-the-ghosts.py
class Solution:
    def escapeGhosts(self, ghosts: List[List[int]], target: List[int]) -> bool:
        closest_ghost = abs(ghosts[0][0] - target[0]) + abs(ghosts[0][1] - target[1])
        for ghost in ghosts:
            closest_ghost = min(closest_ghost, abs(ghost[0] - target[0]) + abs(ghost[1] - target[1]))
        return closest_ghost > abs(target[0]) + abs(target[1])